packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanTreeOffset.js

```diff
@@ -14,33 +14,145 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import cx from 'classnames';
+import _get from 'lodash/get';
+import _find from 'lodash/find';
 import React from 'react';
 import IoChevronRight from 'react-icons/lib/io/chevron-right';
 import IoIosArrowDown from 'react-icons/lib/io/ios-arrow-down';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import { actions } from './duck';
+
+import type { ReduxState } from '../../../types/index';
+import type { Span } from '../../../types/trace';
 
 import './SpanTreeOffset.css';
 
-type SpanTreeOffsetProps = {
-  level: number,
+type SpanTreeOffsetOwnPropsType = {
```

Comment:
What do you think of shortening the name of types that aren't exported? For instance:

- `OwnPropsType`
    - Instead of `SpanTreeOffsetOwnPropsType`
- `PropsType`
    - Instead of `SpanTreeOffsetPropsType`

Reply:
I was following the pattern I saw in other files, such as `SpanBarRow` or `VirtualizedTraceView`
I'm not too particular about either, but I do generally like long-ish variable names for clarity.